### DISCUSSION TITLE
Request: Add Animations and transitions

### DISCUSSION THREAD

**Discussion by Narvalex** [microsoft/fluentui-blazor#580] [Ideas]
Animations can be utilized to provide smooth transitions between UI states, highlight important interactions, and create a more polished look and feel for applications built using this library. This will not only benefit developers but also end-users, as it can significantly improve the overall usability and perceived performance of a Blazor WebAssembly App.

**Comment by vnbaaij** [comment]
We will follow what comes from the Fluent design and should eventually be offered through the web components. See https://fluent2.microsoft.design/motion on which direction this is going.
I don't see us adding sny animations on our own.
